### ISSUE TITLE
Challenging failure under node

### ISSUE THREAD

**Issue by cemerick** [AlexaDeWit/purescript-text-encoding#2]
I happened to end up depending upon this package (via `purescript-arraybuffer`) in the process of attempting some nontrivial JavaScript FFI for the first time, and started getting this error:

```
module.js:550
    throw err;
    ^

Error: Cannot find module 'text-encoding'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/chas/dev/XXX/.psci_modules/node_modules/Data.TextDecoder/foreign.js:6:40)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
```

It took a long while for me to find my way from that error to the README here, and the essential instruction, "If you want to use purescript-text-encoding in such an environment, you will need to install a polyfill: npm install text-encoding."

I'm _very_ new to Purescript, and so I won't venture a PR (and I hope this kind of hidden dependency isn't common among purescript packages! :smile:), but it would at least be nice to wrap the `require("text-encoding")` calls with an exception handler so that a helpful message could be emitted describing the likely situation and probable solution.

**Comment by AlexaDeWit** [comment]
Thanks, that's a great idea! I'll do it probably later today!

**Comment by AlexaDeWit** [comment]
Feedback is resolved now as a result of #3, and the improvements can be found in version `0.0.9`

Thanks!
